core/src/main/java/com/datastax/oss/driver/api/core/CqlSession.java

```diff
@@ -38,6 +38,9 @@ static CqlSessionBuilder builder() {
   /**
    * Executes a CQL statement synchronously (the calling thread blocks until the result becomes
    * available).
+   *
+   * @param statement The statement to execute; never {@code null}.
```

Comment:
If we annotate the method as `@Nonnull`, is it necessary to document it?  Doesn't the annotate show up in the javadocs?

Reply:
Yes it does show up. This is a leftover from a previous state where I was duplicating the information everywhere in the docs. I will remove, thanks for pointing that out.